report-service/src/main/java/gov/cdc/dataingestion/config/ClassConfig.java

```diff
@@ -16,7 +15,7 @@ public class ClassConfig {
 
     @Bean
     IHL7v2Validator hl7v2Validator() {
-        return new HL7v2Validator(new DefaultHapiContext());
+        return new HL7v2Validator();
```

Comment:
can we use spring IoC instead of new?

Reply:
same, , I think we should create new branch off main and fix these as part of clean up